packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -328,7 +328,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
     }
 
     private getChartContainer() {
-        const grid = new TimeGraphChartGrid('timeGraphGrid', this.props.style.rowHeight, this.props.style.lineColor);
+        const grid = new TimeGraphChartGrid('timeGraphGrid', this.props.style.rowHeight, this.props.backgroundTheme === 'light' ? 0xdddddd : 0x34383C);
```

Comment:
I think we should keep the original code here and update the TimeAxisComponent instead to have the previous color. Please open an issue tracker for providing 2 different colors for line and text for the TimeAxisComponent. So, that we can configure the color of the TimeAxisComponent as well.